anchor/programs/depo/src/states/modules/timelock.rs

```diff
@@ -0,0 +1,13 @@
+use anchor_lang::prelude::*;
+
+#[account]
+#[derive(InitSpace)]
+pub struct Timelock {
+  pub release_after: u64,
```

Comment:
Maybe add a comment to specify more precisely what it is (a unix timestamp in seconds ?)